src/store/bookingPortal/index.js

```diff
@@ -166,8 +173,8 @@ const actions = {
     },
 
     // Update Search Text
-    updateSearchText({ commit }, text) {
-        commit('set-search-text', text);
+    updateSearchText({ commit, state }) {
```

Comment:
whats the difference in SearchText and bookingID
updateSearchText and setBookingID are doing similar thing
do we need both?